### DISCUSSION TITLE
necesito ayuda en vue

### DISCUSSION THREAD

**Discussion by miguelapol** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#516] [Q&A] [answered]
Me sale este error descargue la carpeta para avanzar pero tengo este error no me deja iniciar el npm run serve
![vuex](https://user-images.githubusercontent.com/33709574/160050170-d2135c02-6569-4343-b351-36ac2fba850a.png)

**Comment by chistoperez** [comment]
creo que no estás en la carpeta correcta, intenta entrar una carpeta mas a donde se encuentra el proyecto

**Comment by oscardanielM** [comment]
Es porque no encuentra la ruta del package.json, eso es porque no estas dentro de la carpeta que creaste para vue

**Comment by ChristianDeM** [comment]
pon `cd/nombre de tu carpeta` y luego   `npm run serve`

**Comment by GloriaCoralCerecedo** [comment]
Entiendo, descargaste la carpeta donde subió el MC lo que se realizo en VUEX, yo tampoco pude correrlo me parece porque el MC solo subió lo que se realizo más no los documentos extra que no se tocan para que funcione

**Comment by JoseAnayaR** [comment, marked as answer]
Que tal hermano, si aun entrando a la carpeta como los compañeros ya lo comentaron con el cd/nombredetucarpeta tienes el mismo problema que yo, pensé que seria por cuestión de extensiones e instale la siguiente y así funciono 
![image](https://user-images.githubusercontent.com/79932287/160181442-e2b16fd8-fca2-4b46-b36a-92d926a77d4c.png)

**Reply by miguelapol** [reply]
muchas gracias

**Comment by miguelapol** [comment]
muchas gracias a todos compañeros
